labs-ios-starter/Application/Extensions/UIImageView+ImageDownloader.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2020 Shawn James. All rights reserved.
+// UIImageView+ImageDownloader.swift
+
+import UIKit
+
+extension UIImageView {
+    func downloaded(from url: URL, contentMode mode: UIView.ContentMode = .scaleAspectFill) {
```

Comment:
Let's try to use the central code for everything we can to keep things clean. NetworkService can clean this up some, and we make sure to handle errors the same across the app too